products/sle/main.pm

```diff
@@ -1086,6 +1086,10 @@ else {
         loadtest "kiwi_images_test/login_reboot";
         loadtest "kiwi_images_test/validate_build";
     }
+    elsif (get_var("IBM")) {
```

Comment:
don't like this variable . we don't have features created for some certain company . also in current implementation I don't see why not just 
```suggestion
    elsif (check_var('FADUMP', 'basic')) {
```
because if you will define `IBM` but not define FADUMP you will just waste time , right ?

Reply:
There will be more tests in this section,  but I can introduce switch next time. So I will change it.